tests/proxystore_test.py

```diff
@@ -38,8 +38,13 @@ def test_store_from_args(
     for key, value in kwargs.items():
         setattr(args, key, value)
 
-    with mock.patch('psbench.proxystore.init_store'), mock.patch(
-        'psbench.proxystore.GlobusEndpoints.from_json',
+    with (
+        mock.patch('psbench.proxystore.register_store'),
+        mock.patch('psbench.proxystore.FileStore'),
+        mock.patch('psbench.proxystore.RedisStore'),
+        mock.patch('psbench.proxystore.EndpointStore'),
+        mock.patch('psbench.proxystore.GlobusStore'),
+        mock.patch('psbench.proxystore.GlobusEndpoints.from_json'),
```

Comment:
Will this code work with Python 3.7? I think this `with` syntax is newer and you may need to do:
```python
with mock.patch(
    'psbench.proxystore.register_store',
), mock.patch(
    'psbench.proxystore.FileStore',
), mock.patch(
    ...
):
```